lib/manager/jsonnet-bundler/readme.md

```diff
@@ -0,0 +1,5 @@
+Extracts dependencies from `jsonnetfile.json` files, updates `jsonnetfile.lock.json` and updates the `vendor` directory.
+
+Supports [lock file maintenance](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance).
+
+This plugin requires `jsonnet-bundler >= v0.4.0` since previous versions don't support updating single dependencies.
```

Comment:
The documentation says you should use `jsonnet-bundler` equal or greater than `v0.4.0`.

I searched for the string `0.4.0` in the changed files, and get no hits.

Should we have code on the Renovate side that checks the version of `jsonnet-bundler`, or can we rely on the bundler itself to throw an error when you use a unsupported version?

Reply:
@HonkingGoose see [this](https://github.com/renovatebot/renovate/pull/12720/files/05546caad318812c35bc76b594cb5273f7233ce2#diff-bfabdbd781a38490cf7b487876d0c3de22611b32dbf1ff471ef51cbc5c976d3eR40) review for context.

jb 0.4.0 was added in buildpack, but now that I think of it, if you're not runnig renovate with the docker binarySource you will get an error like this (jb 0.2.0 in this case):
```
❯ ./jb-darwin-amd64 update https://github.com/thanos-io/thanos/jsonnet/mixin
Error parsing commandline arguments: unexpected https://github.com/thanos-io/thanos/jsonnet/mixin
jb-darwin-amd64: error: unexpected https://github.com/thanos-io/thanos/jsonnet/mixin
```

I can add a version check if this is something you think this should be handled more gracefully. Is there another manager I could draw inspiration from?

Reply:
We don't usually check for the minimum supported versions of binaries when binarySource=global, so I don't think we need to do this here either